Packs/Orca/pack_metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Orca",
+    "description": "Orca Security cloud integration",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "support@orca.security",
+    "created": "2020-12-20T17:38:25Z",
+    "categories": [
+        "Vulnerability Management"
+    ],
+    "tags": [],
+    "useCases": [],
+    "keywords": []
+}
```

Comment:
In order to be notified of any changes to your pack, please add the key - `githubUser` to this metadata file with your Github User.

Reply:
done